Configuration.props

```diff
@@ -12,7 +12,12 @@
       Condition=" Exists('$(MSBuildThisFileDirectory)Configuration.OperatingSystem.props') And '$(DoNotLoadOSProperties)' != 'True' "
   />
   <PropertyGroup>
+    <ConfigurationPropsImported>true</ConfigurationPropsImported>
     <ProductVersion>10.2.99</ProductVersion>
+    <!-- TF and output paths for MSBuild task projects -->
+    <TargetFrameworkNETStandard>netstandard2.0</TargetFrameworkNETStandard>
+    <SupportingBuildTasksOutputRoot>$(MSBuildThisFileDirectory)bin\Build$(Configuration)\</SupportingBuildTasksOutputRoot>
+    <SupportingBuildTasksDirectory>$(SupportingBuildTasksOutputRoot)$(TargetFrameworkNETStandard)\</SupportingBuildTasksDirectory>
```

Comment:
So this makes our bootstap tasks output to `bin\BuildDebug\netstandard2.0\`?

Would it have been easier to define `<AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>`?

If that is in `Configuration.props`, you probably wouldn't have to change as much in every project.

Reply:
I'd rather not set `AppendTargetFrameworkToOutputPath` in `Configuration.props` just in case we introduce any multi targeting projects or don't want this behavior in projects which import those props for other reasons. We could apply this to the supporting build task projects to simplify this PR, though I'd like to continue using a property which points to the location of `xa-prep-tasks.dll` and `Xamarin.Android.Bootstrap.Tasks.dll` so that we don't have to worry about mass path updates in the future.